cmd/ef_tests/levm/runner/revm_runner.rs

```diff
@@ -101,11 +101,13 @@ pub fn prepare_revm_for_tx<'state>(
     let chain_spec = initial_state
         .chain_config()
         .map_err(|err| EFTestRunnerError::VMInitializationFailed(err.to_string()))?;
+
+    let gas_limit: U256 = test.env.current_gas_limit.into();
     let block_env = RevmBlockEnv {
         number: RevmU256::from_limbs(test.env.current_number.0),
         coinbase: RevmAddress(test.env.current_coinbase.0.into()),
         timestamp: RevmU256::from_limbs(test.env.current_timestamp.0),
-        gas_limit: RevmU256::from_limbs(test.env.current_gas_limit.0),
+        gas_limit: RevmU256::from_limbs(gas_limit.0),
```

Comment:
Isn't there here a straightforward `::from(u64)` method? If so you can avoid having a `gas_limit` variable. Otherwise, I suggest renaming that to `block_gas_limit` to avoid confusion.